packages/repmgr13/repmgr.spec

```diff
@@ -41,9 +45,19 @@ Requires(postun):   initscripts
 Group:        Applications/Databases
 BuildRoot:        %{_tmppath}/%{name}-%{version}%{extra_version}-%{release}-root-%(%{__id_u} -n)
 %endif
-BuildRequires:    postgresql, postgresql-devel, postgresql-static
-BuildRequires:    libxslt-devel, pam-devel, openssl-devel, readline-devel
-Requires:         postgresql-server
+BuildRequires:    postgresql, postgresql-server-devel, postgresql-static
+BuildRequires:    libxslt-devel, pam-devel, openssl-devel, readline-devel flex
+Requires:    postgresql-server
+
+%if 0%{?pgpackageversion} >= 11 && 0%{?pgpackageversion} < 90 && 0%{?rhel} && 0%{?rhel} == 7
+BuildRequires:  llvm-toolset-7
+BuildRequires:  llvm5.0
+%endif
+
+%if 0%{?pgpackageversion} >= 11 && 0%{?pgpackageversion} < 90 && 0%{?rhel} && 0%{?rhel} == 8
```

Comment:
Do we need a conditional for RHEL 9 also?

Reply:
Good 👁️

Reply:
I won't do this with us staying on el8 for now.